### ISSUE TITLE
HP:0004059 - synonym mismatch

### ISSUE THREAD

**Issue by tudorgroza** [obophenotype/human-phenotype-ontology#4976]
**HPO term**
HP:0004059 - `Radial club hand`

**Bug description**
 * Label reads: `Radial club hand` and terms is a `Radial deviation of the hand`
 * EXACT Synonym reads: `Radial dysplasia` - seems to be missing locality or is broader than the term.

**Comment by pnrobinson** [comment]
Actually, these are synonyms.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5805276/
However, this means that we need to merge an existing term Dysplastic radii.
https://hpo.jax.org/app/browse/term/HP:0006433
with this term.
